legend-pure-store/legend-pure-store-relational/legend-pure-runtime-java-extension-compiled-store-relational/pom.xml

```diff
@@ -147,10 +147,6 @@
             <groupId>org.finos.legend.pure</groupId>
             <artifactId>legend-pure-runtime-java-extension-compiled-dsl-mapping</artifactId>
         </dependency>
-        <dependency>
```

Comment:
As discussed with @kevin-m-knight-gs  this is due dependency for `Root_meta_relational_metamodel_Database_Impl `
changing from `Root_meta_relational_metamodel_Database_Impl extends Root_meta_pure_store_set_SetBasedStore`
to 
`Root_meta_relational_metamodel_Database_Impl extends ReflectiveCoreInstance`
due to multiple inheritance. 
Expected